hugo/content/add_genome/recommendations_for_file_formats.md

```diff
@@ -0,0 +1,155 @@
+
+#### Organelle assemblies
+
+It is also possible to display separate assemblies from organelles in the genome browser, such as mitochondria and chloroplasts. In order to display organelle assemblies, we require that they have a gene annotation. Please note that we do not display just organelles in Genome Portal: there also has to be a primary genome assembly from the same species. The same file format recommendations as for primary genome assemblies apply: FASTA files that are preferably gzip-compressed.
+
+### Data tracks
+
+In the context of the genome browser, all other compatible data that describe the assembly is referred to as data tracks. These include annotation of protein coding genes, repeats, tRNA, ncRNA, transcriptome assemblies, to name a few. Some data types tend to come in a specific format (such as VCF for nucleotide variants), whereas some data types can come in several different formats. The Genome Portal does not enforce the use over a certain format over another for the data tracks as long as the format is supported by JBrowse 2. Researchers can thus to a large extent use to use their domain-specific conventions when choosing data format.
+
+Unlike genome assemblies, it is possible to include multiple versions of a data track in the Genome Portal if desired. For instance, a research group might have a version of the Protein Coding Genes track that uses in-house annotations of the gene names that differ from naming format of the version of the track available at NCBI. For cases such as these, we recommend that both tracks are included since this increases clarity and facilitates comparison between the track versions.
+
+In the subsections below we have collected specific recommendations for some commonly used data track file formats.
+
+#### GFF (and GTF)
+
+One of the most common formats for storing annotations for genomic features is GFF (General Feature Format). NCBI for instance uses this for Protein Coding Genes. JBrowse 2 specifically supports the GFF3 version, and older GFF formats thus need to be converted to GFF3 as described below.
+
+GTF is another format for storing annotations that is very similar format to GFF. However, JBrowse 2 does support GTF files and they therefore also need to be converted to GFF3.
+
+There are several bioinformatics tools that can convert GTF and older GFF versions to GFF3. We recommend using <a href="https://agat.readthedocs.io">AGAT</a>. This is a command line toolkit designed by the NBIS bioinformatics platform at SciLifeLab. The AGAT script `agat_convert_sp_gxf2gxf.pl` is used to achieve conversions from GFF and GTF to GFF3.
+
+```text
```

Comment:
This reason why !important is required is because of "text" being added to each code block. I think the easiest work around would be to remove "text" from the multi-line code blocks, and then !important can be removed. 

As "text" doesn't add any benefit as far as I can see?

Reply:
Nice catch, thanks! This explains a lot of the quirks I ran into earlier today :)

The reason I added "text" was that markdownlint-fix was complaining:
`MD040/fenced-code-language Fenced code blocks should have a language specified [Context: "```"]`.

I agree that it is better to avoid !imporant (especially since the fix was so simple). Will also update .markdownlint.yaml so that it skips the "fenced-code-language" check.